### ISSUE TITLE
First Letters spiral recipe has fitted with no winding-placing input since #1651

### ISSUE THREAD

**Issue by ttendoscopie-creator** [ScrollPrize/villa#1716]
**In one sentence:** the `FIT_SPIRAL_CONFIG_OVERRIDES` block published in *From CT Scan to Ancient Text: A First Letters Prize Workflow* (18 August) does not set `input_use_tracks`, and #1651 flipped that default to `False` on 30 August, so following the post on a Grand-Prize-eligible scroll now fits with nothing that can place a point on a winding.

**I was trying to:** fit a spiral on PHerc0826 by following the published First Letters workflow, to see what the pipeline gives on an eligible scroll.

**Using:** `main` at `23adee047`, rebased and re-checked on `b218ace8`. PHerc0826, volume `20250821151701`, the published `surface-m7-L0-th0.2` tracks from `dl.ash2txt.org/datasets/spiral_datasets/`, and the umbilicus published under `representations/umbilicus/` on 3 September. Band z ∈ [11000, 12000), 30,000 iterations, seed 1, one RTX 3090, about 30 minutes per run.

```
FIT_SPIRAL_CONFIG_OVERRIDES='{"z_begin":11000,"z_end":12000,"input_disable_patches":true,
 "loss_weight_shell_outer":0,"loss_weight_shell_patch_radius":0,
 "dense_spacing_mode":"grad_mag","loss_weight_dense_spacing":0}' \
  python fit_spiral.py --dataset <root>
```

**What happened:** the tracks the dataset declares were never loaded and never mentioned, and the fit ran to completion anyway.

```
skipping all verified/unverified patch loading
pcls: 0 cross-patch, 0 unattached
theta topology ready (0.0s, peak RSS 1.02 GiB)
step 0: loss = 2.2, umbilicus = 1.6, sym_dirichlet = 0.6, min_spacing = 0.0
```

There is no `track_radius` term. The post says the fit's geometric evidence is "umbilicus coordinates, extracted tracks, and X/Y surface-normal volumes"; two of those three are gone. Tracks are off by the new default, and normals give orientation rather than winding — two adjacent windings have parallel normals — so nothing is left that can say which winding a point is on. None of the 13 eligible scrolls ships verified patches.

**What I expected or needed:** either the recipe sets `input_use_tracks`, or the post says that tracks are no longer part of the evidence. Right now it names them and does not use them.

**Evidence / reproduction:** the same scroll, same band, same seed, same published umbilicus, one config key apart.

| | recipe as published | + `"input_use_tracks": true` |
|---|---:|---:|
| tracks loaded | 0 | 480,117 |
| exact crossing events | 0 | 1,278,944 |
| `dr_per_winding` | 16.8663 vx (0.158 mm) | 14.8851 vx (0.139 mm) |
| satisfied tracks | 9.7% | **12.6%** |
| satisfied track points | 20.9% | **41.6%** |

With the key set, the loading block appears and the loss changes character:

```
loading tracks from .../PHerc0826_20250821151701_surface_m7_L0_th0.2.dbm
loaded 480117 tracks within z-roi [11000, 12000)
track crossings: 1278944 exact crossing events, 309632/480117 tracks have partners
step 0: loss = 8474.0, ..., track_radius = 8471.8
```

- [x] I personally encountered or reproduced this using the version and data stated above.

## Details

The sharpest number is not in the table above. We first ran the no-tracks fit against an umbilicus of our own making, which turned out to sit 234–464 voxels from the published one. Its `dr_per_winding` came out **16.8650**; against the published umbilicus, **16.8663**. Moving the axis by 464 voxels — around 25 windings at the fitted spacing — changes the fitted winding spacing by **0.0013 voxels**.

With no tracks the fit is not reading the scroll. `dr` settles where the regularisers put it, and both no-track runs converge smoothly to a *lower* final loss than the supervised one (0.1 against ~300), because there is almost nothing left to fit. That is what made this hard to notice: every number a reader would check looks better without the tracks.

Track satisfaction is quoted because `fit_spiral` reports it, not as a measure of correctness — per #1621 the metric is periodic in the winding, so a surface displaced by a whole number of windings scores identically. It is a radial agreement, and it still halves.

Suggested fixes, cheapest first:

1. Add `"input_use_tracks": true` to the overrides in the post, and a line in `38_tutorial_spiral.md`.
2. #1713 (open) makes `fit_spiral` report which declared inputs it discarded and warn when no winding-placing input is left, so the next default change does not fail silently.
3. If tracks genuinely are of no further use on scrolls that ship them, the post is the place to say so.

#1651's note is "disable track losses since these are no longer useful", which is reasonable on PHercParis4 — it ships `verified_patches/`, `outer_shell/`, `eval_fibers/`, `lasagna_inputs/` and `tracks/`. The published spiral datasets for the 13 eligible scrolls are `tracks/` and nothing else for nine of them, and empty for PHerc1203, PHerc1218, PHerc1447 and PHerc1545.

Not verified: whether the same holds on a scroll with a verified-patch pack; how many people have run the recipe since 30 August; `spiral_service.py`, which resolves inputs on its own path.

**Comment by ttendoscopie-creator** [comment]
Small update, in case it helps triage rather than as a ping.

The docs half of this is now moving: #1719 adds the `input_use_*` note to the spiral tutorial, and I have just trimmed it per @pmh47's review — enable-inputs text only, no issue references, nothing prize-specific. That covers the tutorial.

What this issue reports is still open on the other side. The `FIT_SPIRAL_CONFIG_OVERRIDES` block published in the First Letters workflow post (18 August) does not set `input_use_tracks`, so following the post as written still fits with no winding-placing input on a scroll whose only published supervision is tracks. Either that block gains the flag, or the sentence naming extracted tracks as part of the fit's geometric evidence no longer holds for it.

If the post lives in this repo, point me at the file and I will send a one-line patch. If the recipe is deliberately meant to run without tracks, I am glad to be told so and will close this.

**Comment by ttendoscopie-creator** [comment]
Now that #1719 is merged, the tutorial side of this is covered. The one remaining point is the First Letters workflow post of 18 August, which as far as I can tell lives outside this repo, so there's nothing I can send a patch for. If leaving `input_use_tracks` out of that post's override block is intended, feel free to close this.

**Comment by bnleft** [comment]
Confirming on a second scroll, and adding one number for whoever updates the post.

PHerc0211, z 10000–11000, the 18 Aug post's overrides verbatim plus `input_use_tracks: true`, villa `2dcfaf6a`:

```
loaded track crossing cache .../PHerc0211_20250821151803_surface_m7_L0_th0.2.dbm.crossings.npz: 14360311 tracks, 63368856 directed records
loaded packed track store .../PHerc0211_20250821151803_surface_m7_L0_th0.2.dbm.vctracks: 741046 tracks, 34978733 points in ROI
loaded 741046 tracks within z-roi [10000, 11000)
```

Without the key, nothing about tracks is printed and the fit runs on umbilicus + normals only, as #1716 describes. A second thing the post predates, which cost me a fit attempt: `lasagna_data.py` on current main only loads `normal_x`/`normal_y`/`gradient_magnitude` through `pack_resident_pools.py` sidecars ("there is no other loading path"); the error is `lasagna normals: resident-pool sidecar '...respool_g2_pair' not found`. I have a short README addition for that in a separate PR.

Run and logs: https://github.com/bnleft/first-light-pherc0211 (`logs/2026-09-17-fit_ACW_1500.log`).

AI-assisted (Claude Code), human-directed.
